apps/web/src/components/Publication/Actions/Collect/CollectModule.tsx

```diff
@@ -194,57 +188,18 @@ const CollectModule: FC<CollectModuleProps> = ({ count, setCount, publication, e
     onError
   });
 
-  const [createCollectProxyAction, { loading: proxyActionLoading }] = useProxyActionMutation({
-    onCompleted,
-    onError
-  });
-
-  const createViaProxyAction = async (variables: any) => {
-    const { data } = await createCollectProxyAction({ variables });
-    if (!data?.proxyAction) {
-      await createCollectTypedData({
-        variables: {
-          request: { publicationId: publication?.id },
-          options: { overrideSigNonce: userSigNonce }
-        }
-      });
-    }
-  };
-
   const createCollect = async () => {
     if (!currentProfile) {
       return toast.error(Errors.SignWallet);
     }
 
     try {
-      if (isFreeCollectModule && !collectModule?.followerOnly) {
```

Comment:
I'm not sure what features we are potentially breaking by removing these conditions?